server/plugin.go

```diff
@@ -86,6 +86,13 @@ func (p *Plugin) OnActivate() error {
 	if err != nil {
 		return errors.Wrap(err, "failed to initialize own command handling")
 	}
+
+	if err := p.api.Admin.SynchronizeApps(); err != nil {
+		mm.Log.Error("Can't synchronize", "err", err.Error())
+		// TODO should we fail or continue?
```

Comment:
I think we should continue, but we need to figure out the monitoring piece that let's us know quickly which apps failed to synchronize (without sifting through the log of error and info entries).